pkg/skyenv/values.go

```diff
@@ -170,3 +173,44 @@ func HomePath() string {
 	dir, _ := os.UserHomeDir() //nolint
 	return dir
 }
+
+// Config returns either UserConfig or PackageConfig based on permissions
+func Config() PkgConfig {
+	if IsRoot() {
```

Comment:
I dont understand why we are returning different configs based on permissions. It sounds like we assume that if the binary is run as root, we are dealing with a package install, which may be the case reasonably often, but will frustrate anyone not using the package install with root privileges also running the VPN client with root.

Reply:
The permission tells where files should be written to, basically. So you don't want to have the visor use a config with paths that it either can't write to or shouldn't write too

This was basically existing logic that i moved from a few different places to here and it's not universally implemented, only where appropriate.